src/gravel/controllers/user.py

```diff
@@ -0,0 +1,74 @@
+# project aquarium's backend
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+import bcrypt
+import json
+
+from pydantic import BaseModel, Field
+from typing import List, Optional
+
+from gravel.controllers.kv import KV
+
+
+class CredentialsModel(BaseModel):
+	username: str = Field("", title="The user name")
+	password: str = Field("", title="The password in plain text")
+
+
+class UserModel(BaseModel):
+	username: str = Field("", title="The user name")
+	password: str = Field("", title="The password")
+	full_name: str = Field("", title="The full name of the user")
+	disabled: Optional[bool] = Field(False, title="Is the user disabled?")
```

Comment:
if we're setting the default as `False`, I don't see a reason to keep this as `Optional`. If the field is not specified, it will take the default value.